misc/taskmetadata-validator/Makefile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+.PHONY: all clean docker image
+
+all: taskmetadata-validator image
+
+#VENDOR_DIR=$(shell echo $(cd ../../agent/vendor; pwd))
+VENDOR_DIR=$(shell echo $(cd ../../agent/vendor; pwd))
+taskmetadata-validator: taskmetadata-validator.go
```

Comment:
I'm not an makefile expert, is it normal to put a file as dependency?

Reply:
it's the whole premise of makefiles :)